doc/howto_release.md

```diff
@@ -70,6 +70,12 @@ Modify the VERSION file use the dedicated script, for RC1, e.g.:
 ./utils/update_version.py rc 1
 ```
 
+Note: to change the version after the RC cycle to an official release, run
```

Comment:
Let's do three subsections like in the case of post-release update or just specify these clearly as two options. One is example the other is a note. (We have manual pages with very short "description" and everything there is a "note". Let's not fall into that here.)

> For release candidates, use `./utils/update_version.py rc`, e.g. for RC1:
>
> ./utils/update_version.py rc 1

> For final releases, use:
>
> ./utils/update_version.py release